migtests/tests/resumption/import-file/single-large-table/config.yaml

```diff
@@ -14,7 +14,7 @@ row_count:
 
 # Resumption Settings
 resumption:
-  max_restarts: 30
+  max_restarts: 20
```

Comment:
is this just for testing or changing it in the PR (if so, any reason?)?

Reply:
The test takes almost 5 hours, so had a discussion with Shubham and decided to reduce it. 
@shubham-yb can you pls review this change?

Reply:
Can we also increase the time for each run in between restarts so that the load is a bit more balanced.